src/components/controllers/page/sidebar-nav.module.less

```diff
@@ -19,11 +19,13 @@
 	display: block;
 	position: relative;
 	padding: 0.25rem 0.75em 0.25rem 1em;
-	transition: font-weight 0.3s;
+	transition: background-color 0.2s, box-shadow 0.2s, color 0.2s;
 	color: var(--color-sidebar-link);
 
 	&:hover {
-		background-color: fade(@color-brand, 50%);
+		background-color: fade(@color-brand, 10%);
+		box-shadow: inset 0 0 0 1px fade(@color-brand, 10%);
+		color: #000;
```

Comment:
If we want to change the text color on hover we need to also account for users with a preference for dark themes. This is hard to read:

![temp](https://user-images.githubusercontent.com/33403762/187806649-2d4b957b-64a5-41ec-91e5-f899b537069d.PNG)

Reply:
heh, yeah just noticed that (my color scheme changes at 9pm)